assets/src/sass/utilities/_theme-supports.scss

```diff
@@ -14,33 +14,33 @@ $color-palette:
 	blue            $wmui-color-blue            $wmui-color-base100 $wmui-color-blue90,
 	blue-50         $wmui-color-blue50          $wmui-color-base100 $wmui-color-blue90,
 	blue-90         $wmui-color-blue90          $wmui-color-base0   $wmui-color-blue50,
-	blue70          $wmui-color-blue70          $wmui-color-base0   $wmui-color-base50,
+	blue-70          $wmui-color-blue70          $wmui-color-base0   $wmui-color-base50,
```

Comment:
@pamprn09 Would changing these color variable names break styling of existing blocks on the site? If so, is there a way to check if these theme colors are being used anywhere on the site?

Reply:
@goldenapples those changes to fix color blocks that were not working.